openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -302,11 +302,24 @@ For more details and concrete mitigations, see (#security-considerations-pre-aut
 
 # Credential Offer Endpoint {#credential-offer-endpoint}
 
-This endpoint is used by a Credential Issuer that is already interacting with an End-User who wishes to initiate a Credential issuance. It is used to pass available information relevant for the Credential issuance to ensure a convenient and secure process.
+Credential Offer Endpoint is used by a Credential Issuer that is already interacting with an End-User who wishes to initiate a Credential issuance. It is used to pass available information relevant for the Credential issuance to ensure a convenient and secure process as defined in (#credential-offer).
```

Comment:
```suggestion
Wallet exposes the Credential Offer Endpoint, enabling the Credential Issuer to pass the Credential Offer to the Wallet, thereby initiating the Credential Issuance process. In this way, the Credential Issuer, which an End-User interacts with, uses Credential Offer to pass to the Wallet information necessary for convenient and secure Credential Issuance process defined in (#credential-offer) .
```

I would apply this change and in the line below indicated that there are several strategies Credential Offer Endpoint strategies like link, custom scheme, qr code  and so on.